index.html

```diff
@@ -9256,6 +9256,7 @@ <h5>Presentational Role Inheritance</h5>
 			<rdef>textbox</rdef>
 			<div class="role-description">
 				<p>A type of input that allows free-form text as its value.</p>
+				<p>Authors MUST limit its children to completely presentational elements (like a spacer image, decorative graphic, or clearing element).</p>
```

Comment:
Another example couple be a decorative icon corresponding to the purpose of the textbox, like the little portrait silhouette that sometimes accompanies a “Name” or “Username” field?